src/bacnet/version.h

```diff
@@ -15,8 +15,8 @@
 #define BACNET_VERSION(x, y, z) (((x) << 16) + ((y) << 8) + (z))
 #endif
 
-#define BACNET_VERSION_TEXT "1.4.1"
-#define BACNET_VERSION_CODE BACNET_VERSION(1, 4, 1)
+#define BACNET_VERSION_TEXT "1.4.2"
```

Comment:
We should change the version in a separate Pull Request by changing the version and updating CHANGELOG.txt.